extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -276,29 +282,56 @@ export class InstrumentationService {
   }
 
   /**
+   * @param {!Element} el
+   * @private
+   * @return {!Object<string, string>}
+   */
+  extractAnalyticsVariables_(el) {
+    const dataSet = el.dataset;
+    const analyticsVariables = {};
+    for (const dataKey in dataSet) {
+      if (dataKey.indexOf(VARS) === 0) {
+        let variableName = dataKey.replace(VARS, '');
+        variableName = variableName[0].toLowerCase() + variableName.slice(1);
```

Comment:
element.dataSet convers everything that follows data into camel case. since 'v' in vars is the first letter, 'v' in variables is capital.

This is the transformation this code is doing:

`data-vars-variable-name` -> `varsVariableName` -> `VariableName` -> `variableName`